next.clayui.com/content/docs/components/navigation/breadcrumbs.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: "Breadcrumbs"
+description: "Breadcrumb is a secondary navigation pattern that identifies the page position inside a hierarchy."
+packageNpm: "@clayui/navigation"
+packageVersion: "3.0.0-milestone.3"
+packageStatus: "Beta"
+sibling: "docs/components/navigation/css-breadcrumbs.html"
+---
+
+import {Breadcrumbs} from '../../../../src/components/clay/Breadcrumb';
+
+<div class="nav-toc-absolute">
+<div class="nav-toc">
+
+- [API](#api)
+
+</div>
+</div>
+
+<div class="clay-site-alert alert alert-info">
+	Check the <a href="https://liferay.design/lexicon/">Lexicon</a> <a href="https://liferay.design/lexicon/core-components/alerts/">Alerts Pattern</a> for a more in-depth look at the motivations and proper usage of this component.
```

Comment:
`<a href="https://liferay.design/lexicon/core-components/alerts/">Alerts Pattern</a>`

Is this correct?

Reply:
nope 😂